apps/site/api/tests/presentation_tests.py

```diff
@@ -10,12 +10,12 @@
 class ApiPresentationTest(object):
     name = 'New Presentation'
     description = 'description of presentation'
-    tags = "a, b, c"
+    tags = [u'a',u'b',u'c']
```

Comment:
I'm really confused by this. Why for some tests are there arrays and others strings? I think this test needs to be developed.